### PR TITLE
Default option rules

### PR DESCRIPTION
**Pull request by roman-simionov**

### CODE REVIEW

**Review by pavelgruba — DISMISSED** [review]

File: js/core/options/utils.js [hunk 1]
```diff
@@ -27,3 +27,7 @@ export const deviceMatch = (device, filter) => isEmptyObject(filter) || findBest
 export const getFieldName = fullName => fullName.substr(fullName.lastIndexOf('.') + 1);
 
 export const getParentName = fullName => fullName.substr(0, fullName.lastIndexOf('.'));
+
+export default function createDefaultRules(options = []) {
+    return options;
+}
```
Comment (MaximKudriavtsev) [inline js/core/options/utils.js]: I think we should not combine `old` and `new` styles. Use arrow functions

File: testing/jest/button.tests.tsx [hunk 2]
```diff
@@ -414,4 +440,43 @@ describe('Button', () => {
 
         expect(tree.is('.dx-button')).toBe(true);
     });
+
+    describe('DefaultOptionRules', () => {
+        function getDefaultProps() {
```
Comment (MaximKudriavtsev) [inline testing/jest/button.tests.tsx]: arrow function

File: testing/jest/button.tests.tsx [hunk 3]
```diff
@@ -414,4 +440,43 @@ describe('Button', () => {
...
+            defaultOptions({
+                device: () => false,
+                options: {},
+            });
+            return Button.defaultProps;
```
Comment (MaximKudriavtsev) [inline testing/jest/button.tests.tsx:450]: It's very bed using global functions and global variables. Can we rewrite it to pure function?

Comment (roman-simionov) [reply testing/jest/button.tests.tsx:450]: No, We can't

**Review by pavelgruba — APPROVED** [review]

**Review by MaximKudriavtsev — APPROVED** [review]

**Pull request merged**